src/main/java/org/moditect/kcctl/service/compaction/KafkaDatabaseHistoryCompaction.java

```diff
@@ -0,0 +1,252 @@
+package org.moditect.kcctl.service.compaction;
+
+import io.debezium.relational.history.DatabaseHistory;
+import io.debezium.relational.history.DatabaseHistoryException;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+import io.debezium.document.DocumentReader;
+
+import io.debezium.util.Collect;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.function.Consumer;
+
+public class KafkaDatabaseHistoryCompaction extends AbstractDatabaseHistoryCompaction {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KafkaDatabaseHistoryCompaction.class);
+
+    private static final String CONSUMER_PREFIX = CONFIGURATION_FIELD_PREFIX_STRING + "consumer.";
+    private static final String PRODUCER_PREFIX = CONFIGURATION_FIELD_PREFIX_STRING + "producer.";
+
+    public static final Field BOOTSTRAP_SERVERS = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "kafka.bootstrap.servers")
```

Comment:
Let's avoid this notion of field definitions, as they are specific to Kafka Connect APIs really. The bootstrap server is part of the `Context` already (see here: https://github.com/kcctl/kcctl/blob/main/src/main/java/org/moditect/kcctl/service/Context.java#L39). kcctl works with predefined named _environments_ between a user can switch. The other command-specific options like polling intervall should be parameters of the actual command itself.